shared/js/notification_helper.js

```diff
@@ -30,26 +30,29 @@
     // an object -> {raw: string}
     send: function nh_send(titleL10n, options) {
       return new Promise(function(resolve, reject) {
-        navigator.mozL10n.once(function() {
-          var title = getL10n(titleL10n);
+        navigator.mozL10n.once(() => {
+          getL10n(titleL10n).then(title => {
+            options.title = title;
+            if (options.bodyL10n) {
+              return getL10n(options.bodyL10n).then(body => {
+                options.body = body;
+              });
+            }
+          }).then(() => {
+            options.dir = navigator.mozL10n.language.direction;
+            options.lang = navigator.mozL10n.language.code;
```

Comment:
This got me thinking:  in l20n.js, `L10n.languages` is also a promise. Should we land something similar in l10n.js to make it easier to write forward-compatible code?